app/views/pipeline_outputs/show.html.erb

```diff
@@ -50,6 +52,16 @@
           <tr>
             <td><%= element.tax_id %></td>
             <td><%= element.name %></td>
+            <td><%= case element.tax_level
+                   when 1; 'species'
```

Comment:
I will probably define
TAX_LEVEL_SPECIES  = 1
TAX_LEVEL_SPECIES = 2 
in the taxid_count model.

Reply:
cool